website/docs/r/kvstore_backup_policy.html.markdown

```diff
@@ -0,0 +1,45 @@
+
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_kvstore_backup_policy"
+sidebar_current: "docs-alicloud-resource-kvstore-backup_policy"
+description: |-
+  Provides a backup policy for ApsaraDB Redis / Memcache instance resource.
+---
+
+# alicloud\_kvstore\_backup_policy
+
+Provides a backup policy for ApsaraDB Redis / Memcache instance resource. 
+
+## Example Usage
+
+```
+resource "alicloud_kvstore_backup_policy" "redisbackup" {
+  instance_id             = "${alicloud_kvstore_instance.myredis.id}"
+  preferred_backup_time   = "00:00Z-04:00Z"
+  preferred_backup_period = "Friday"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+* `instance_id` - (Required) The id of ApsaraDB for Redis or Memcache intance.
+* `preferred_backup_time`- (Required) Backup time, in the format of HH:mmZ- HH:mm Z
+* `preferred_backup_period` - (Required) Backup Cycle. Allowed values: Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday
+
+## Attributes Reference
+
+The following attributes are exported:
+* `id` - The id of the backup policy.
```

Comment:
please add a black between line 33 and 34.

Reply:
done